### PR TITLE
Rename to spetlr

### PR DESCRIPTION
**Pull request by mrmasterplan**
change name to spetlr

### CODE REVIEW

**Review by JeppeBlixen — DISMISSED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# Before creating a release
+# Version
 
-- Make sure you increment the `__version__`, otherwise the release pipeline will fail.
+- The pipeline will automtically increment the patch version
+- increment the minor version when adding a new feature
+- the major version should only be incremented if and only if features are removed.
```
Comment (LauJohansson) [inline .github/CONTRIBUTING.md:5]: What about backward compatability?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# atc-dataplatform
-A common set of python libraries for DataBricks. 
+# spetlr
+A common set of python libraries for Databricks.
```
Comment (LauJohansson) [inline README.md]: "A python SPark ETL libRary (SPETLR) for Databricks"

Comment (LauJohansson) [reply README.md]: "A python spark ETL library (SPETLR) for Databricks"

File: setup.cfg [hunk 3]
```diff
@@ -1,8 +1,8 @@
 [metadata]
-name = atc-dataplatform
+name = spetlr
 author = ATC.Net
```
Comment (LauJohansson) [inline setup.cfg]: Author=ATC-Net?

File: setup.cfg [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 version = file: src/VERSION.txt
-description = A common set of python libraries for DataBricks
+description = A common set of python libraries for Production ready Spark ETL
```
Comment (LauJohansson) [inline setup.cfg]: "A python spark ETL library (SPETLR) for Databricks"

Comment (LauJohansson) [reply setup.cfg]: or use this description in the README.md also...?

File: src/spetlr/__init__.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+"""
+A common set of python libraries for spark etl.
+See https://github.com/spetlr/spetlr for details
```
Comment (LauJohansson) [inline src/spetlr/__init__.py]: Lets make a common phrase to use all places.

File: src/spetlr/orchestrators/eh2bronze/EhToDeltaBronzeTransformer.py [hunk 6]
```diff
@@ -88,9 +87,7 @@ def process(self, df: DataFrame) -> DataFrame:
         )
 
         # Add streaming time
-        streaming_time = datetime.datetime.now(datetime.timezone.utc).replace(
-            microsecond=0
-        )
+        streaming_time = dt_utc().replace(microsecond=0)
```
Comment (LauJohansson) [inline src/spetlr/orchestrators/eh2bronze/EhToDeltaBronzeTransformer.py]: The transformer should not be dependent of the spetlrtools - right?

This change is wrong

File: tests/cluster/eh/test_eh_saving.py [hunk 7]
```diff
@@ -2,21 +2,21 @@
...
+from spetlr.eh import EventHubCapture, EventHubJsonPublisher
+from spetlr.eh.EventHubCaptureExtractor import EventHubCaptureExtractor
+from spetlr.etl import Transformer
+from spetlr.functions import init_dbutils
+from spetlr.orchestrators import EhJsonToDeltaOrchestrator
+from spetlr.spark import Spark
 from tests.cluster.values import resourceName
 
-from .AtcEh import AtcEh
+from .SpetlrEh import SpetlrEh
```
Comment (LauJohansson) [inline tests/cluster/eh/test_eh_saving.py:19]: consider using absolute path like the rest of the imports?

File: README.md (hunk 2 above)
Comment (mrmasterplan) [reply README.md]: I like it.

File: setup.cfg (hunk 3 above)
Comment (LauJohansson) [reply setup.cfg]: Spetlr-org?

Comment (LauJohansson) [reply setup.cfg]: Maybe it conflict a bit when it is under atc org....

File: src/spetlr/__init__.py (hunk 5 above)
Comment (LauJohansson) [reply src/spetlr/__init__.py]: "A python spark ETL library (SPETLR) for Databricks"

**Comment by LauJohansson** [comment]
Let's get this PR approved.
It blocks the integration tests of the rest of the PRs (due to renaming of the cosmos db - free tier issue)

File: src/spetlr/orchestrators/eh2bronze/EhToDeltaBronzeTransformer.py (hunk 6 above)
Comment (mrmasterplan) [reply src/spetlr/orchestrators/eh2bronze/EhToDeltaBronzeTransformer.py]: Well spotted. Thank you

File: tests/cluster/eh/test_eh_saving.py (hunk 7 above)
Comment (mrmasterplan) [reply tests/cluster/eh/test_eh_saving.py:19]: let's do this later

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (mrmasterplan) [reply .github/CONTRIBUTING.md:5]: this is a question for another time

**Review by LauJohansson — APPROVED** [review]

**Review by LauJohansson — CHANGES_REQUESTED** [review]
Do not change name.
We need to be able to support changes to atc-dataplatform.

Therefore keeping the atc-daplatform as-is - and use the name in another setting.

**Pull request closed without merging**